CommonPythonCTest.cmake

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2016, Juan Hernando <juan.hernando@fi.upm.es>
+#
+# Creates a per project pythontests target.
+# This target creates a test for each .py file in the current
+# directory tree. The per project targett is included into the global
+# 'tests' from CommonCTest.cmake.
+#
+# Input:
+# * PYTHON_TEST_DEPENDENCIES targets that are required by the tests
+# * EXCLUDE_FROM_TESTS relative paths to test files to exclude; optional
```

Comment:
PYTHON_TEST_EXCLUDE for consistency in this file at least? We have to touch the CPP one as well at some point...